source/expressions.tex

```diff
@@ -3929,11 +3929,13 @@
 When a prvalue \tcode{v} of object pointer type is converted to
 the object pointer type ``pointer to \cv{}~\tcode{T}'', the result is \tcode{static_cast<\cv{} T*>(static_cast<\cv{}~void*>(v))}.
 \begin{note}
-Converting a prvalue of type ``pointer to \tcode{T1}'' to
-the type ``pointer to \tcode{T2}'' (where \tcode{T1} and \tcode{T2} are
-object types and where the alignment requirements of \tcode{T2} are no
-stricter than those of \tcode{T1}) and back to its original type yields
-the original pointer value.
+Converting a pointer of type ``pointer to \tcode{T1}''
+that points to an object of type \tcode{T1}
+to the type ``pointer to \tcode{T2}'',
```

Comment:
I still prefer the parentheses in place of the comma. I find ending up with a comma before "and" to be unfortunate when the instance of "and" is not being used to join independent clauses into a sentence.